File: argentina.py
n = 234612846789231
adn =["A"]
def newValue(char):
  switch = {
    "A": "AL",
    "L": "PACA",
    "P": "CP",
    "C": "PC"
  }
  return switch[char]

def replace(adn):
  newAdn = []
  newAdn = adn.copy()
  for x in range(0, len(adn)):
    newAdn[x] = newValue(adn[x])
  return list("".join(newAdn))

for x in range(0, n):
  adn = replace(adn)

print(adn)

File: argentina.js
const n = 234612846789231;
let adn = ["A"];

function newValue(char) {
  switch (char) {
    case "A":
      return "AL";
    case "L":
      return "PACA";
    case "P":
      return "CP";
    case "C":
      return "PC";
  }
}
function replace(adn) {
  let newAdn = [];
  for (let i = 0; i < adn.length; i++) {
    newAdn[i] = newValue(adn[i]);
  }
  return newAdn.join("").split("");
}

for (let i = 0; i < n; i++) {
  adn = replace(adn);
}
console.log(adn);
